lib/app/modules/home/controllers/home_controller.dart

```diff
@@ -528,7 +528,7 @@ class HomeController extends GetxController {
       await IsarDb.deleteAlarm(alarm.isarId);
     }
 
-    // Display snackbar
+    Get.back();
```

Comment:
Might be a better idea to use GetX to check if snackbars are active using `isSnackbarOpen` and close it using `closeAllSnackBars`